src/Microsoft.DotNet.Interactive.SqlServer.Tests/MsSqlConnectionTests.cs

```diff
@@ -126,5 +126,44 @@ public async Task Field_types_are_deserialized_correctly()
                 row["database_id"].Should().BeOfType<int>();
             }
         }
+
+        [MsSqlFact]
+        public async Task Empty_results_are_displayed_correctly()
+        {
+            var connectionString = MsSqlFact.GetConnectionStringForTests();
+            using var kernel = await CreateKernel();
+            var result = await kernel.SubmitCodeAsync(
+                             $"#!connect --kernel-name adventureworks mssql \"{connectionString}\"");
+
+            result.KernelEvents
+                  .ToSubscribedList()
+                  .Should()
+                  .NotContainErrors();
+
+            result = await kernel.SubmitCodeAsync($@"
+#!adventureworks --mime-type {TabularDataFormatter.MimeType}
+use tempdb;
+create table dbo.EmptyTable(column1 int, column2 int, column3 int);
+select * from dbo.EmptyTable;
+drop table dbo.EmptyTable;
+");
+
+            var events = result.KernelEvents.ToSubscribedList();
+
+            events.Should().NotContainErrors();
+
+            var value = events.Should()
+                              .ContainSingle<DisplayedValueProduced>()
+                              .Which;
+
+            var tables = (IEnumerable<IEnumerable<IEnumerable<(string, object)>>>) value.Value;
+            var table = tables.Single().ToTable();
+
+            table.Count.Should().Be(1);
+            table[0].Count.Should().Be(3);
```

Comment:
I don't think we should be creating fake data in order to display the Data Explorer. This data might be displayed or shared in other ways and the record count should be accurate.

Reply:
Are we able to display only columns (without data) in the data explorer? That's the behavior in other SQL query editors like SSMS and ADS. Otherwise, I can just display a message that no rows were returned.

Reply:
I don't know. A simple message would be fine for now. A nicer display for schema info might be nice to have but we can discuss later.